versionutils/git/version_test.go

```diff
@@ -0,0 +1,21 @@
+package git_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/solo-io/go-utils/versionutils/git"
+)
+
+var _ = Describe("Git Versions", func() {
+
+	It("works as expected", func() {
+		refInfo, err := git.GetGitRefInfo("./../..")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(refInfo).NotTo(BeNil())
+		// Git commit sha is always 40 characters long
+		Expect(refInfo.Hash).To(HaveLen(40))
+		// Our tags always start with"v"
+		Expect(refInfo.Tag).To(HavePrefix("v"))
+		// Nothing reliable to test the branch with, unfortunately
```

Comment:
I kind of just want to make sure it doesn't crash because you are array explicitly indexing, does that make sense?

Reply:
Removed this test, but added a check for the array length